clang/include/clang/Basic/FileManager.h

```diff
@@ -292,21 +292,21 @@ class FileManager : public RefCountedBase<FileManager> {
   /// MemoryBuffer if successful, otherwise returning null.
   llvm::ErrorOr<std::unique_ptr<llvm::MemoryBuffer>>
   getBufferForFile(FileEntryRef Entry, bool isVolatile = false,
-                   bool RequiresNullTerminator = true,
+                   bool RequiresNullTerminator = true, bool IsText = true,
```

Comment:
can you add some comments explaining what `IsText` is about and when it should be set to `false`?

also can you make it the last parameter ?

Reply:
I've added a brief comment here as well, please let me know if its sufficient or needs more, thanks!